src/supermarket/app/views/application/_search.html.erb

```diff
@@ -46,7 +46,7 @@
     <h4> Select Supported Platforms </h4>
     <div class = "advanced_search_platforms" >
 
-      <% %w(aix amazon centos debian fedora freebsd gentoo mac_os_x omnios openbsd oracle redhat ubuntu scientific smartos solaris suse windows zlinux).each do |platform| %>
+      <% %w(aix amazon centos debian fedora freebsd gentoo mac_os_x omnios openbsd opensuse opensuseleap oracle redhat ubuntu scientific smartos solaris suse windows zlinux).each do |platform| %>
```

Comment:
I feel like this list could be centralized somewhere, but that's for another time.